project/src/com/google/daggerquery/executor/QueryExecutorMain.java

```diff
@@ -0,0 +1,30 @@
+/*
+Copyright 2020 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+ */
+
+package com.google.daggerquery.executor;
+
+import com.google.daggerquery.executor.services.InternalSourcesLoader;
+import com.google.daggerquery.executor.services.DaggerQueryController;
+
+public class QueryExecutorMain {
+  public static void main(String[] args) {
+    new DaggerQueryController().execute(
+        /*arguments = */    args,
+        /*printStream = */  System.out,
+        /*sourcesLoader =*/ new InternalSourcesLoader()
```

Comment:
Nit: would you mind making this consistent (eg. adding a space to the sourcesLoader comment or removing a space from the two others?

Reply:
Sure! Thanks 😊